src/Modules/Visualization/ShowFieldGlyphs.cc

```diff
@@ -342,161 +404,195 @@ void GlyphBuilder::renderVectors(
 
   GlyphGeom glyphs;
   auto facade(field->mesh()->getFacade());
-
-  //Temporary fix for cloud field data until after IBBM
-  bool done = false;
-  // Render cell data
-  if (!finfo.is_linear())
+  
+  bool normalizeGlyphs = state->getValue(ShowFieldGlyphs::NormalizeGlyphs).toBool();
+  bool renderBidirectionaly = state->getValue(ShowFieldGlyphs::RenderBidirectionaly).toBool();
+  bool renderGlphysBellowThreshold = state->getValue(ShowFieldGlyphs::RenderGlyphsBellowThreshold).toBool();
+  float threshold = state->getValue(ShowFieldGlyphs::Threshold).toDouble();
+
+  //sets feild location for consant feild data 1: node centered 2: edge centered 3: face centered 4: cell centered
+  int feildLocation = finfo.is_point()*1 + finfo.is_line()*2 + finfo.is_surface()*3 + finfo.is_volume()*4;
```

Comment:
I think there's a better way to do this--there ought to be an enum value you can query and then switch on, that tells you the data location. I will check what it is.

Reply:
I'm thinking it's `field->vfield()->basis_order()`, but that might not quite be what you need. We'll merge as is (after you fix the `feild/field` typos) and we can revisit after.